app/src/components/governance/PageProposals.vue

```diff
@@ -378,6 +378,7 @@ h4 {
   border-radius: 0.25rem;
   width: 100%;
   margin: 0 0.5rem;
+  white-space: nowrap;
```

Comment:
what does this look like on mobile?

Reply:
After another fix like this:

![image](https://user-images.githubusercontent.com/40721795/93783520-0de39280-fc2c-11ea-9daf-742364ade20d.png)